packages/server/postgres/utils/checkEqBase.ts

```diff
@@ -1,7 +1,97 @@
 import {RTable, TableSchema} from '../../database/stricterR'
-import areEqual from 'fbjs/lib/areEqual'
 import getPg from '../getPg'
 
+export function areEqual(a, b): boolean {
+  if (a === b) return true
+  const aType = typeof a
+  const bType = typeof b
+  // eslint-disable-next-line eqeqeq
+  if (aType !== bType) return a == b // to handle cases like wrapped types for primitives
+  switch (aType) {
+    case 'string':
+    case 'bigint':
+    case 'number':
+    case 'symbol':
+    case 'undefined':
+    case 'function':
+    case 'boolean':
+      return a === b
+    case 'number':
+      return (Number.isNaN(a) && Number.isNaN(b)) || a === b
+    case 'object':
+      if (a === null || b === null) return a === b
+
+      let result = areEqualArray(a, b)
+      if (result !== undefined) return result
+
+      result = areEqualDate(a, b)
+      if (result !== undefined) return result
+
+      result = areEqualRegExp(a, b)
+      if (result !== undefined) return result
+
+      return areEqualObject(a, b)
+    default:
+      // ts check that all types are handled
+      const _: never = aType
+      return a === b
+  }
+}
+
+function areEqualDate(a, b): boolean | undefined {
+  const isADate = a instanceof Date
+  const isBDate = b instanceof Date
+  if (isADate && isBDate) {
+    return +a === +b
+  } else if (isADate || isBDate) {
+    return false
+  }
+}
+
+function areEqualRegExp(a: RegExp, b: RegExp): boolean | undefined {
+  const isARegExp = a instanceof RegExp
+  const isBRegExp = b instanceof RegExp
+  if (isARegExp && isBRegExp) {
+    return (
+      a.source === b.source &&
+      a.global === b.global &&
+      a.multiline === b.multiline &&
+      a.ignoreCase === b.ignoreCase
+    )
+  } else if (isARegExp || isBRegExp) {
+    return false
+  }
```

Comment:
-1 I get warnings when running the test. You need to explicitly `return undefined` at the end, same in other functions